packages/react-components/README.md

```diff
@@ -13,32 +13,42 @@ Getting set up is as easy as using the `AgoricProvider`. Under the hood, it uses
 ```tsx
 import { AgoricProvider, ConnectWalletButton } from '@agoric/react-components';
 import { wallets } from 'cosmos-kit';
+// Only needed if customizing the wallet connection modal, or if using `NetworkDropdown`.
+import { ThemeProvider, useTheme } from '@interchain-ui/react';
 import '@agoric/react-components/dist/style.css';
 
 const App = () => {
+  const { themeClass } = useTheme();
   return (
-    <AgoricProvider
-      wallets={wallets.extension}
-      defaultNetworkConfig={{
-        // testChain is optional, defaulting to Agoric mainnet otherwise.
-        testChain: {
-          chainId: 'agoriclocal',
-          chainName: 'agoric-local',
-        },
-        apis: {
-          rest: ['http://localhost:1317'],
-          rpc: ['http://localhost:26657'],
-        },
-      }}
-    >
-      <ConnectWalletButton />
-    </AgoricProvider>
+    <ThemeProvider>
+      <div className={themeClass}>
+        <AgoricProvider
+          wallets={wallets.extension}
+          defaultNetworkConfig={{
+            // testChain is optional, defaulting to Agoric mainnet otherwise.
+            testChain: {
+              chainId: 'agoriclocal',
+              chainName: 'agoric-local',
+            },
```

Comment:
nit: is this nesting necessary? Maybe just make `chainId` and `chainName` optional, noting they will be pulled from chain-registry if not provided?

Reply:
I guess the benefit of nesting is:
- You can couple the properties so that you either need both or neither
- It makes it clearer that they're used for test chains rather than mainnet

I suppose there would be a way with ts to do the coupling but I'm not sure if it would be as neat. I lean towards keeping the nesting but just a preference.